KerbalStuff/config.py

```diff
@@ -15,13 +15,27 @@ def get_env_var_or_config(section, key):
     if env_var:
         return env_var
     else:
-        return config.get(section, key)
+        return config.get(section, key, fallback=None)
 
 
-_cfg = lambda k: get_env_var_or_config(env, k)
-_cfgi = lambda k: int(_cfg(k))
-_cfgb = lambda k: strtobool(_cfg(k)) == 1
-_cfgl = lambda k: ast.literal_eval(_cfg(k))
+def _cfg(k):
+    return get_env_var_or_config(env, k)
+
+
+def _cfgi(k):
+    val = _cfg(k)
+    return int(val) if val is not None else 0
+
+
+def _cfgb(k):
+    val = _cfg(k)
+    return strtobool(val) == 1 if val is not None else False
```

Comment:
Same suggestion: return None if the value is missing.
Accessors shouldn't imply default values IMO.
```suggestion
    return strtobool(val) == 1 if val is not None else None
```

Reply:
Same response, returning `None` doesn't suit the purpose of this function or improve how it works. `False` is a sensible default for a bool config setting.